controllers/apply/forms/reaching-communities-form.js

```diff
@@ -133,7 +192,39 @@ formModel.registerReviewStep({
 
 formModel.registerSuccessStep({
     title: 'We Have Received Your Idea',
-    processor: function() {}
+    processor: function(formData) {
+        return new Promise((resolve, reject) => {
+            let summary = formModel.getStepsWithValues(formData);
+            let flatData = formModel.getStepValuesFlattened(formData);
+            const dateNow = moment().format('dddd, MMMM Do YYYY, h:mm:ss a');
+
+            let to = `${flatData['first-name']} ${flatData['first-name']} <${flatData['email']}>`;
+
+            // generate HTML for email
+            app.render(
+                'emails/applicationSummary',
+                {
+                    summary: summary,
+                    form: formModel,
+                    data: flatData
+                },
+                (err, html) => {
+                    if (err) {
+                        reject(err);
+                    }
+                    mail
+                        .send({
+                            html: html,
+                            sendTo: to,
+                            sendFrom: 'Big Lottery Fund <noreply@blf.digital>',
+                            subject: `Your Reaching Communities application - ${dateNow}`
+                        })
+                        .catch(err => reject(err))
+                        .then(resolve(formData));
```

Comment:
Is this the same as `.then(() => resolve(formData))`. Does the syntax you have here end up trying to pass the result of resolve onto something?